<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ContactDialog</class>
 <widget class="QDialog" name="ContactDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>594</width>
    <height>447</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Contact</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="lbFullName">
           <property name="text">
            <string>Full Name</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="leFullName"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="gbNames">
         <property name="title">
          <string>Name</string>
         </property>
         <layout class="QGridLayout" name="layNames">
          <item row="0" column="0">
           <widget class="QLineEdit" name="leName1"/>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="leName2"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="gbPhones">
         <property name="title">
          <string>Phone</string>
         </property>
         <layout class="QGridLayout" name="layPhones">
          <item row="0" column="0">
           <widget class="QLineEdit" name="lePhone1"/>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="cbPhoneType1">
            <property name="minimumSize">
             <size>
              <width>128</width>
              <height>0</height>
             </size>
            </property>
            <property name="maxVisibleItems">
             <number>32</number>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QToolButton" name="btnDelPhone1">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="doublecontact.qrc">
              <normaloff>:/img/16x16/del.png</normaloff>:/img/16x16/del.png</iconset>
            </property>
            <property name="arrowType">
             <enum>Qt::NoArrow</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="gbEmails">
         <property name="title">
          <string>Email</string>
         </property>
         <layout class="QGridLayout" name="layEmails">
          <item row="0" column="0">
           <widget class="QLineEdit" name="leEmail1"/>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="cbEmailType1">
            <property name="minimumSize">
             <size>
              <width>128</width>
              <height>0</height>
             </size>
            </property>
            <property name="maxVisibleItems">
             <number>32</number>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QToolButton" name="btnDelEmail1">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="doublecontact.qrc">
              <normaloff>:/img/16x16/del.png</normaloff>:/img/16x16/del.png</iconset>
            </property>
            <property name="arrowType">
             <enum>Qt::NoArrow</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="btnAdd">
           <property name="text">
            <string>Add...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cbAdd">
           <item>
            <property name="text">
             <string>name</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>phone</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>e-mail</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QFrame" name="frameForFuture">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="labelForFuture">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>204</y>
          <width>121</width>
          <height>91</height>
         </rect>
        </property>
        <property name="text">
         <string>For Future</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="cbBirthday">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>9</y>
          <width>91</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Birthday</string>
        </property>
       </widget>
       <widget class="QDateTimeEdit" name="dteBirthday">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>40</y>
          <width>194</width>
          <height>23</height>
         </rect>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="doublecontact.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ContactDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ContactDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
